extension/models/catalog/rootCategories/rootCategories.js

```diff
@@ -0,0 +1,28 @@
+const Category = require('../category/category')
```

Comment:
Please change to ES6 style import

Reply:
As discussed, import is only experimental, therefor we'll keep 'require' for now